src/main/java/entities/Mealplan.java

```diff
@@ -1,30 +1,31 @@
 package entities;
 
+import java.io.Serializable;
 import java.util.ArrayList;
 
-public class Mealplan {
+public class Mealplan implements Serializable {
 
-    ArrayList<ArrayList<Recipe>> mealplan;
+    ArrayList<ArrayList<String>> mealplan;
 
     public Mealplan(){
         this.mealplan = new ArrayList<>();
         int i = 0;
         while(i<3){
-            mealplan.add(new ArrayList<Recipe>());
+            mealplan.add(new ArrayList<>());
             i++;
         }
     }
 
-    public void addMealPlan(Recipe r, int meal){
```

Comment:
Even though we have java doc, I think we could give this parameter a more meaningful name for readers to understand.